www/js/ui.js

```diff
@@ -618,7 +618,7 @@ $("#shuffleplaylist").click(function() {
 /* load channel */
 
 var loc = document.location+"";
-var m = loc.match(/\/r\/([a-zA-Z0-9-_]+)/);
+var m = loc.match(/\/\w+\/([a-zA-Z0-9-_]+)/);
```

Comment:
Actually, I thought of a way to make all that totally obsolete. The channel template already knows the channels name. It can deposit it in the template and make life elegant.